misc/dist/linux/org.godotengine.Godot.desktop

```
@@ -5,12 +5,14 @@ GenericName[el]=Ελεύθερη μηχανή παιχνιδιού
 GenericName[fr]=Moteur de jeu libre
 GenericName[nl]=Libre game-engine
 GenericName[ru]=Свободный игровой движок
+GenericName[uk_UA]=Вільний ігровий рушій
 GenericName[zh_CN]=自由的游戏引擎
 Comment=Multi-platform 2D and 3D game engine with a feature-rich editor
 Comment[el]=2D και 3D μηχανή παιχνιδιού πολλαπλών πλατφορμών με επεξεργαστή πλούσιο σε χαρακτηριστικά
 Comment[fr]=Moteur de jeu 2D et 3D multiplateforme avec un éditeur riche en fonctionnalités
 Comment[nl]=Multi-platform 2D- en 3D-game-engine met een veelzijdige editor
 Comment[ru]=Кроссплатформенный движок с многофункциональным редактором для 2D- и 3D-игр
+Comment[uk_UA]=Багатофункціональний кросплатформний рушій для створення 2D та 3D ігор
```

I believe the country code shouldn't necessary here. It's necessary for `zh_CN` because there are multiple variants of Chinese based on the country, but (AFAIK) for Ukrainian it should be unified?

The main difference is that e.g. a Ukrainian dev living in France may have a locale like `uk_FR` which wouldn't match. If the language code is only `uk`, it would be a match.
```suggestion
GenericName[uk]=Вільний ігровий рушій
GenericName[zh_CN]=自由的游戏引擎
Comment=Multi-platform 2D and 3D game engine with a feature-rich editor
Comment[el]=2D και 3D μηχανή παιχνιδιού πολλαπλών πλατφορμών με επεξεργαστή πλούσιο σε χαρακτηριστικά
Comment[fr]=Moteur de jeu 2D et 3D multiplateforme avec un éditeur riche en fonctionnalités
Comment[nl]=Multi-platform 2D- en 3D-game-engine met een veelzijdige editor
Comment[ru]=Кроссплатформенный движок с многофункциональным редактором для 2D- и 3D-игр
Comment[uk]=Багатофункціональний кросплатформний рушій для створення 2D та 3D ігор
```